galaxy_ng/app/settings.py

```diff
@@ -40,10 +40,6 @@
 GALAXY_API_STAGING_DISTRIBUTION_BASE_PATH = "staging"
 GALAXY_API_REJECTED_DISTRIBUTION_BASE_PATH = "rejected"
 
-# The format for the name of the per account synclist, and the
-# associated repository, distribution, and distribution base_paths
-GALAXY_API_SYNCLIST_NAME_FORMAT = "{account_name}-synclist"
```

Comment:
This is still used in auth.py, so we want to keep this setting. We do need to make sure auth.py `_ensure_synclists` does not call `_ensure_repository` which creates a synclist repo

Reply:
We also need to remove `_ensure_repository` from auth/auth.py so we do not create synclist repos anymore

It would also be good to do a general sweep of the code to make sure we are not looking up synclist repos anywhere